### DISCUSSION TITLE
time left

### DISCUSSION THREAD

**Discussion by SWRJEEDOM** [arkypita/LaserGRBL#2475] [Ideas]
Thank you for this wonderfull tool.

I'd love to get the time left in the progression bar "click" : time left...new "click" time from the begining

![Capture](https://github.com/user-attachments/assets/0cb58d89-9581-4346-aa42-af9278392a3e)

**Comment by Tribo-2dot0** [comment]
You can calculate based on the estimated time...
![Captura de tela 2024-12-23 001946](https://github.com/user-attachments/assets/0aa38816-30f6-4656-b90f-06c3a61684a9)
